tests/end-to-end/external-solution-integration/pkg/step/retry.go

```diff
@@ -21,6 +21,12 @@ func (r *Retried) Name() string {
 }
 
 func (r *Retried) Run() error {
+	retryOptions := []retry.Option{
+		retry.Attempts(20), // at max (100 * (1 << 13)) / 1000 = 819,2 sec
```

Comment:
The default value of one minute should be more than sufficient here. FI the event hasn't arrived for a minute it means the system is broken. We cannot wait for 10 minutes.

Reply:
Removed and increased the num of attempts for checkcounterpod